### DISCUSSION TITLE
"we weren't able to approve your educational"  How can I get verified for Student

### DISCUSSION THREAD

**Discussion by youssef-elfakir** [community/community#51955] [GitHub Education]
I sent a request for github student pack
But I'm not sure why I refused.
Is this rejection final?
I think the problem is in the picture of the academic affiliation document , And how can I resend the picture?

![Screenshot 2023-04-04 141036](https://user-images.githubusercontent.com/129417034/229822272-f88000cf-63d1-4749-9759-a534385e6706.png)
![d868d978-0359-4a87-8dc6-42b0c107be91](https://user-images.githubusercontent.com/129417034/229822452-28a6496f-a8fb-498f-af2d-20b20f303ddf.jpg)

**Comment by Ntshangase** [comment]
Hi @youssef-elfakir 

Your registration was successful and you should be able to enjoy the student benefits at this time. The re-verification should be done after a year.

**Reply by youssef-elfakir** [reply]
thank you
